### ISSUE TITLE
make-tree example only has root

### ISSUE THREAD

**Issue by cmears** [deech/fltkhs#11]
I am not sure what the make-tree example is supposed to do, but all I get is the single root item:

![make-tree](https://cloud.githubusercontent.com/assets/955423/7904453/abe8264a-083e-11e5-830a-3eb936e69650.png)

The "test" items don't appear.

**Comment by deech** [comment]
You're right, I'm seeing the same behavior. Please see `tree-simple` for a better demonstration of the tree widget. Thanks for the bug report!
